### PR TITLE
Add camel-google-calendar wrapper

### PR DESCRIPTION
**Pull request by ozgurkisir-qlik**
Add camel-google-calendar wrapper

### CODE REVIEW

File: features/src/main/feature/camel-features.xml [hunk 1]
```diff
@@ -975,6 +975,23 @@
 <!--        <bundle dependency='true'>wrap:mvn:org.threeten/threetenbp/${google-cloud-threetenbp-version}</bundle>-->
 <!--        <bundle>mvn:org.apache.camel/camel-google-bigquery/${project.version}</bundle>-->
 <!--    </feature>-->
+    <feature name='camel-google-calendar' version='${project.version}' start-level='50'>
+        <feature version='${camel.osgi.version.range}'>camel-core</feature>
+        <feature version="[33,34)">guava</feature>
+        <feature version='[4,5)'>http-client</feature>
+        <bundle dependency='true'>mvn:com.google.api-client/google-api-client/2.4.0</bundle>
```
Comment (f2par0) [inline features/src/main/feature/camel-features.xml]:
```suggestion
        <bundle dependency='true'>mvn:com.google.api-client/google-api-client/${google-api-client-version}</bundle>
```

File: features/src/main/feature/camel-features.xml [hunk 2]
```diff
@@ -975,6 +975,23 @@
...
+        <bundle dependency='true'>wrap:mvn:com.google.http-client/google-http-client/${google-cloud-http-client-version}$overwrite=merge&amp;Import-Package=com.google.common*;version="[33,34)"</bundle>
+        <bundle dependency='true'>wrap:mvn:com.google.auth/google-auth-library-credentials/${google-auth-library-oauth2-http-version}</bundle>
+        <bundle dependency='true'>wrap:mvn:com.google.auth/google-auth-library-oauth2-http/${google-auth-library-oauth2-http-version}</bundle>
+        <bundle dependency='true'>wrap:mvn:io.opencensus/opencensus-api/0.31.1</bundle>
```
Comment (f2par0) [inline features/src/main/feature/camel-features.xml:949]:
```suggestion
        <bundle dependency='true'>wrap:mvn:io.opencensus/opencensus-api/${google-http-client-opencensus-api-version}</bundle>
```

File: features/src/main/feature/camel-features.xml [hunk 3]
```diff
@@ -975,6 +975,23 @@
...
+        <bundle dependency='true'>wrap:mvn:io.opencensus/opencensus-contrib-http-util/0.31.1</bundle>
```
Comment (f2par0) [inline features/src/main/feature/camel-features.xml:950]:
```suggestion
        <bundle dependency='true'>wrap:mvn:io.opencensus/opencensus-contrib-http-util/${google-http-client-opencensus-api-version}</bundle>
```

File: features/src/main/feature/camel-features.xml [hunk 4]
```diff
@@ -938,6 +938,23 @@
...
+        <bundle dependency='true'>wrap:mvn:io.opencensus/opencensus-api/${google-http-client-opencensus-api-version}</bundle>
+        <bundle dependency='true'>wrap:mvn:io.opencensus/opencensus-contrib-http-util/${google-http-client-opencensus-api-version}</bundle>
+        <bundle dependency='true'>wrap:mvn:com.google.oauth-client/google-oauth-client/${google-oauth-client-version}$overwrite=merge&amp;Import-Package=com.google.common*;version="[33,34)",*</bundle>
+        <bundle dependency='true'>wrap:mvn:com.google.http-client/google-http-client-gson/${google-cloud-http-client-version}</bundle>
+        <bundle dependency='true'>wrap:mvn:com.google.http-client/google-http-client-apache-v2/${google-cloud-http-client-version}</bundle>
+        <bundle dependency='true'>wrap:mvn:com.google.http-client/google-http-client-jackson2/${google-cloud-http-client-version}</bundle>
+        <bundle dependency='true'>wrap:mvn:com.google.apis/google-api-services-calendar/${google-api-services-calendar-version}</bundle>
+        <bundle>mvn:org.apache.camel.karaf/camel-google-calendar/${project.version}</bundle>
+    </feature>
```
Comment (essobedo) [inline features/src/main/feature/camel-features.xml]:
```suggestion
    <feature name='camel-google-calendar' version='${project.version}' start-level='50'>
        <feature version='${camel.osgi.version.range}'>camel-core</feature>
        <feature version="[33,34)">guava</feature>
        <feature version='[4,5)'>http-client</feature>
        <bundle dependency='true'>mvn:com.google.api-client/google-api-client/${google-api-client-version}</bundle>
        <bundle dependency='true'>wrap:mvn:com.google.http-client/google-http-client/${google-cloud-http-client-version}$overwrite=merge&amp;Import-Package=com.google.common*;version="[33,34)",*</bundle>
        <bundle dependency='true'>wrap:mvn:com.google.auth/google-auth-library-credentials/${google-auth-library-oauth2-http-version}</bundle>
        <bundle dependency='true'>wrap:mvn:com.google.auth/google-auth-library-oauth2-http/${google-auth-library-oauth2-http-version}</bundle>
        <bundle dependency='true'>wrap:mvn:io.opencensus/opencensus-api/0.31.1</bundle>
        <bundle dependency='true'>wrap:mvn:io.opencensus/opencensus-contrib-http-util/0.31.1</bundle>
        <bundle dependency='true'>wrap:mvn:com.google.oauth-client/google-oauth-client/${google-oauth-client-version}$overwrite=merge&amp;Import-Package=com.google.common*;version="[33,34)",*</bundle>
        <bundle dependency='true'>wrap:mvn:com.google.http-client/google-http-client-gson/${google-cloud-http-client-version}</bundle>
        <bundle dependency='true'>wrap:mvn:com.google.http-client/google-http-client-apache-v2/${google-cloud-http-client-version}</bundle>
        <bundle dependency='true'>wrap:mvn:com.google.http-client/google-http-client-jackson2/${google-cloud-http-client-version}</bundle>
        <bundle dependency='true'>wrap:mvn:com.google.apis/google-api-services-calendar/${google-api-services-calendar-version}</bundle>
        <bundle>mvn:org.apache.camel.karaf/camel-google-calendar/${project.version}</bundle>
    </feature>
```

File: features/src/main/feature/camel-features.xml [hunk 5]
```diff
@@ -938,6 +938,23 @@
...
     </feature>
+    <feature name='camel-google-calendar' version='${project.version}' start-level='50'>
+        <feature version='${camel.osgi.version.range}'>camel-core</feature>
+        <feature version="[33,34)">guava</feature>
+        <feature version='[4,5)'>http-client</feature>
+        <bundle dependency='true'>mvn:com.google.api-client/google-api-client/${google-api-client-version}</bundle>
+        <bundle dependency='true'>wrap:mvn:com.google.http-client/google-http-client/${google-cloud-http-client-version}$overwrite=merge&amp;Import-Package=com.google.common*;version="[33,34)"</bundle>
+        <bundle dependency='true'>wrap:mvn:com.google.auth/google-auth-library-credentials/${google-auth-library-oauth2-http-version}</bundle>
+        <bundle dependency='true'>wrap:mvn:com.google.auth/google-auth-library-oauth2-http/${google-auth-library-oauth2-http-version}</bundle>
+        <bundle dependency='true'>wrap:mvn:io.opencensus/opencensus-api/${google-http-client-opencensus-api-version}</bundle>
```
Comment (f2par0) [inline features/src/main/feature/camel-features.xml]:
```suggestion
         <bundle dependency='true'>wrap:mvn:io.opencensus/opencensus-api/0.31.1</bundle></bundle>
```

File: features/src/main/feature/camel-features.xml [hunk 6]
```diff
@@ -938,6 +938,23 @@
...
+        <bundle dependency='true'>wrap:mvn:io.opencensus/opencensus-contrib-http-util/${google-http-client-opencensus-api-version}</bundle>
```
Comment (f2par0) [inline features/src/main/feature/camel-features.xml]:
```suggestion
        <bundle dependency='true'>wrap:mvn:io.opencensus/opencensus-contrib-http-util/0.31.1</bundle>
```

**Review by f2par0 — APPROVED** [review]
- no conflict :heavy_check_mark: 
- compilation works :heavy_check_mark: 
- no unnecessary usage of wrap protocol :heavy_check_mark: 
- no unnecessary dependencies :heavy_check_mark: 
- no hardcoded version if it's in the parent pom :heavy_check_mark: 
- feature is placed in alphabetical order in the file camel-features.xml  :heavy_check_mark: 
- usage of dependent features (netty, jackson, jakarta ...) :heavy_check_mark:

**Review by essobedo — APPROVED** [review]

**Pull request merged**